exec/r-quboleCommand.py

```diff
@@ -0,0 +1,141 @@
+try:
+    from qds_sdk.qubole import Qubole
+    from qds_sdk.commands import *
+    import qds_sdk.exception
+except ImportError:
+    print """You need to install qds-sdk-py from http://github.com/qubole/qds-sdk-py"""
```

Comment:
Should be ok, it throws an error inside R shell

> library(qds)
Loading required package: rPython
Loading required package: RJSONIO
> results<-qds::quboleHiveCommand(query="show tables")
You need to install qds-sdk-py from http://github.com/qubole/qds-sdk-py
Error in python.exec(python.command) : 
  global name 'Qubole' is not defined

> hive<-HiveCommand$new() 
You need to install qds-sdk-py from http://github.com/qubole/qds-sdk-py
Error in python.exec(python.command) : 
  global name 'Qubole' is not defined
>